week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -9,32 +9,38 @@ To be safe, they need to land on the first unamed planet that has Oxygen levels
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
 */
 
-function safeLevels() {
-
+function safeLevels(arr) {
+  let found = arr.filter(element => element > "19.5%" && element < "23.5%");
+
+  console.log(found);
+  if (found[0] === undefined) {
```

Comment:
This is an interesting check to be doing - can you explain what it's achieving / why it's here?